package.json

```diff
@@ -7,7 +7,7 @@
     "publish": "lerna publish --conventional-commits --message \"chore: publish\" --yes",
     "bundle": "lerna run bundle --stream --concurrency 1",
     "bundle-min": "lerna run bundle --stream --concurrency 1 -- -- --configEnv=prod",
-    "test": "lerna run test --stream --concurrency 1",
+    "test": "npx jest -c scripts/jest/jest.config.js",
```

Comment:
I don't think this needs the `npx` on the start and can we use the long version of the jest flag (i.e. `--config`)

Reply:
Good spot, thanks - that's fixed now. I've also updated to use `--config` instead of `-c`.